include-build/roborazzi-gradle-plugin/src/main/java/io/github/takahirom/roborazzi/GenerateComposePreviewRobolectricTestsExtension.kt

```
@@ -327,3 +330,63 @@ private fun verifyLibraryDependencies(
+        project.configurations
+          .filter { it.isCanBeResolved }
+          .flatMap { config ->
+            config.resolvedConfiguration.resolvedArtifacts
+              .filter { artifact ->
+                artifact.moduleVersion.id.group == "io.github.sergio-sastre.ComposablePreviewScanner" &&
+                artifact.moduleVersion.id.name == "android"
+              }
+              .map { it.moduleVersion.id.version }
+          }
+          .firstOrNull()
+      } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)